### PR TITLE
Padding reduced and margin removed to avoid the clicking issue

### LINKED ISSUE #168: Clicking on icons is not always precise.
**Describe the bug**
For example, sometimes when clicking on the filter icon, the search icon is clicked.

**Cause**
This is due to the very high padding on the individual icons. 
Screenshot from 2020-03-02 08-53-31

**Solution**
Decrease the padding, or remove it altogether. Then the readjustment and realignment of the icons need to be done.

### PR DESCRIPTION
**Pull request by Maxslide**
Fixes #168 

**Description**
Changed the Padding and margin. Margin -> 0 px and padding -> 6%.

**Other information**
Made the padding to be precisely 6%, which causes exactly no overlap

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by makkoncept** [comment]
@Maxslide Please post a screenshot of the icons.

**Review by makkoncept — CHANGES_REQUESTED** [review]
The issue has not been resolved when I run this locally because you have made changes to the social media share icons which appear on the info popup.

**Comment by makkoncept** [comment]
@Maxslide Can you please fix this soon. This is to be shipped with the new update.

**Comment by Maxslide** [comment]
I'll work on this, the changes need to be made to the filer and search icon right? I believe I have done that..I'll recheck

**Comment by makkoncept** [comment]
Filter, search and cross icon.

**Comment by Maxslide** [comment]
I'll just check and try to update it as soon as possible

**Comment by Maxslide** [comment]
Hey, I did make change to the fa class but however on inspect element I don't see any change for the same, I have added the extension from the load unpacked option on Google chrome, is there any further steps I need to take to make sure the changes are reflected, that is for debugging do I need it do any other step?

**Comment by makkoncept** [comment]
You are looking in the wrong file. The correct one is _search.scss

**Comment by Maxslide** [comment]
I did look into this file, i did not find any margin tag for the filter-icon or any other class, i know this is a super simple issue, but idk why i cant find any margin option for any of the buttons, i dont see from where they get the margin they do

**Comment by makkoncept** [comment]
@Maxslide  Do the following change in [_search.scss](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/sass/layout/_search.scss)

**Before**
```css
  i {
    position: absolute;
    right: 0;
    top: 50%;
    transform: translateY(-50%);
    color: $color-secondary-light;

    &#filter-icon {
      right: 6%;
    }

    &.clear-search-button {
      color: #ff4e14;
      right: 12%;

      &:hover {
        color: #fb4102;
      }
    }
    &:hover {
      color: $color-secondary-dark;
      cursor: pointer;
    }
  }
```

**After**
```css
  i {
    position: absolute;
    padding: 0;
    margin: 0;
    top: 50%;
    transform: translateY(-50%);
    color: $color-secondary-light;

    &#search-icon {
      right: 4%;
    }

    &#filter-icon {
      right: 10%;
    }

    &.clear-search-button {
      color: #ff4e14;
      right: 16%;

      &:hover {
        color: #fb4102;
      }
    }
    &:hover {
      color: $color-secondary-dark;
      cursor: pointer;
    }
  }
}
```

**Comment by makkoncept** [comment]
It will work.
Screenshot from 2020-03-04 00-12-53
Screenshot from 2020-03-04 00-12-57

**Comment by Maxslide** [comment]
I did the exact same thing right now, but i see now changes in my extension, i even reloaded teh extension, all i need to do is save the code after my changes and reload teh extension adn view it right?

**Comment by makkoncept** [comment]
@Maxslide You have to build it again.

**Comment by Maxslide** [comment]
Screenshot from 2020-03-04 00-30-50
Screenshot from 2020-03-04 00-32-00

**Comment by Maxslide** [comment]
> @Maxslide You have to build it again.

OHH! MY BAD!

**Comment by Maxslide** [comment]
I have made the changes, kindly review

File: src/popup/sass/components/_popup.scss [hunk 1]
```diff
@@ -115,8 +115,8 @@
 // share buttons
 .fa {
   box-sizing: content-box;
-  padding: 20px;
-  margin-right: 10px;
+  padding: 6px;
+  margin-right: 0px;
```
Comment (makkoncept) [inline src/popup/sass/components/_popup.scss]: Undo these changes.

File: src/popup/sass/layout/_search.scss [hunk 2]
```diff
@@ -25,14 +25,16 @@
     top: 50%;
     transform: translateY(-50%);
```
Comment (makkoncept) [inline src/popup/sass/layout/_search.scss]: You also have to make changes in the `i` rule like making padding 0, etc. Refer to my comment above.

File: src/popup/sass/layout/_search.scss [hunk 3]
```diff
@@ -25,14 +25,16 @@
...
     color: $color-secondary-light;
-
+    &#search-margin{
+      right : 4%;
+    }
```
Comment (makkoncept) [inline src/popup/sass/layout/_search.scss]: - Add space before {

**Review by makkoncept — CHANGES_REQUESTED** [review]

File: src/popup/sass/layout/_search.scss (hunk 2 above)
Comment (Maxslide) [reply src/popup/sass/layout/_search.scss]: Done

File: src/popup/sass/components/_popup.scss (hunk 1 above)
Comment (Maxslide) [reply src/popup/sass/components/_popup.scss]: Done

File: src/popup/sass/layout/_search.scss [hunk 4]
```diff
@@ -22,17 +22,21 @@
   i {
     position: absolute;
     right: 0;
```
Comment (makkoncept) [inline src/popup/sass/layout/_search.scss]: Remove `right: 0`. This was also written in the comment above .

**Review by makkoncept — CHANGES_REQUESTED** [review]

File: src/popup/sass/layout/_search.scss (hunk 4 above)
Comment (Maxslide) [reply src/popup/sass/layout/_search.scss]: Done. I'm really sorry for all the tiny errors i made

File: src/popup/sass/layout/_search.scss [hunk 5]
```diff
@@ -21,18 +21,21 @@
...
-    right: 0;
+    padding: 0;
+    margin: 0;
     top: 50%;
     transform: translateY(-50%);
     color: $color-secondary-light;
-
+    &#search-margin {
+      right : 4%;
+    }
```
Comment (makkoncept) [inline src/popup/sass/layout/_search.scss]: @Maxslide This was not `#search-margin` but `#search-icon`. This was also mentioned in the comment above.
Nevermind, it has been fixed in https://github.com/creativecommons/ccsearch-browser-extension/commit/5ac81e6f402cfe81b79bb45e65b93c45287e56a2
